api/comics/management/commands/data.py

```diff
@@ -0,0 +1,4 @@
+descriptions = ['Gamer','Hercules and the Amazon Women','Pirates of the Caribbean: Dead Mans Chest','Moneyball','M for Mother (Mim mesle madar)','Julie','Loggerheads','Mr. Vampire (Geung si sin sang)','Hamlet, Prince of Denmark','The Lost Prince']
```

Comment:
We want to generate fake text for these, so we have text as long as a real-life description would be, so I'd delete these